### PR TITLE
FlyCheat configureable

### PR DESCRIPTION
**Pull request by goldlife2904**
To enable FlyCheat (Default):
no change in Script is needed as this is default.

To disable FlyCheat set in gameflow.lua:

Flow.EnableFlyCheat(false)

### CODE REVIEW

File: TombEngine/Scripting/Internal/ReservedScriptNames.h [hunk 1]
```diff
@@ -109,6 +109,7 @@ static constexpr char ScriptReserved_EndLevel[]					= "EndLevel";
 static constexpr char ScriptReserved_GetSecretCount[]			= "GetSecretCount";
 static constexpr char ScriptReserved_SetSecretCount[]			= "SetSecretCount";
 static constexpr char ScriptReserved_AddSecret[]				= "AddSecret";
+static constexpr char ScriptReserved_SetFlyCheatEnabled[]		= "SetFlyCheatEnabled";
```
Comment (Sezzary) [inline TombEngine/Scripting/Internal/ReservedScriptNames.h:112]: Why not simply name it `EnableFlyCheat`? :)

Comment (goldlife2904) [reply TombEngine/Scripting/Internal/ReservedScriptNames.h]: Had change it to EnableFlyCheat option. Builders can set in gameflow.lua: Flow.EnableFlyCheat(false)

**Comment by Lwmte** [comment]
I'd recommend to rename function to `ToggleFlyCheat` and also add similar function for `TogglePlayAnyLevel`

**Comment by goldlife2904** [comment]
> I'd recommend to rename function to `ToggleFlyCheat` and also add similar function for `TogglePlayAnyLevel`

Yes, but it will be weird if builders has to write ToggleFlyCheat(false) - so you dont know if this "false" will Toggle "On" or Toggle "Off"... cause this are bool params it will be better to name function with words like "enable" or "disable" - it's clearly more understandable I think

**Pull request merged**